tests/faultydata/variants/duplicated_sampleid/README.md

```diff
@@ -0,0 +1,6 @@
+Data file example_1.vcf.gz was copied from tests/data/variants/example_1, then change sample id S2 to S1
+
+Index file was generated by
+```
```

Comment:
I wasn't sure about this either, just did it anyway. I think many text editors do support syntax highlighting for markdown. so if one views the README with an editor, I think it is quite clear?